wolfssl/wolfcrypt/wc_port.h

```diff
@@ -618,6 +618,7 @@
                 #include <winsock2.h>
             #endif
             #include <windows.h>
+            #include <ws2tcpip.h>
```

Comment:
Should this be wrapped with `!defined(WOLFSSL_USER_IO)`? If this is related to sockets we shouldn't add a new include if using custom read/write IO callbacks.